From alan@linux.intel.com  Wed Jan 27 15:38:05 2010
From: Alan Cox <alan@linux.intel.com>
Date: Mon, 18 Jan 2010 15:32:39 +0000
Subject: Staging: et131x: kill off the FBR_NUM_DES type
To: greg@kroah.com
Message-ID: <20100118153239.6884.4367.stgit@localhost.localdomain>


This is another 10 bit value with the high bits clear, and where the
type doesn't get used anywhere properly anyway

Signed-off-by: Alan Cox <alan@linux.intel.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/et131x/et1310_address_map.h |   12 ------------
 1 file changed, 12 deletions(-)

--- a/drivers/staging/et131x/et1310_address_map.h
+++ b/drivers/staging/et131x/et1310_address_map.h
@@ -451,18 +451,6 @@ typedef union _RXDMA_CSR_t {
  * 31-10: unused
  * 9-0: fbr ndesc
  */
-typedef union _RXDMA_FBR_NUM_DES_t {
-	u32 value;
-	struct {
-#ifdef _BIT_FIELDS_HTOL
-		u32 unused:22;		/* bits 10-31 */
-		u32 fbr_ndesc:10;	/* bits 0-9 */
-#else
-		u32 fbr_ndesc:10;	/* bits 0-9 */
-		u32 unused:22;		/* bits 10-31 */
-#endif
-	} bits;
-} RXDMA_FBR_NUM_DES_t, *PRXDMA_FBR_NUM_DES_t;
 
 /*
  * structure for free buffer ring 0 available offset reg in rxdma address map
